### DISCUSSION TITLE
How to add new app in LTSP client

### DISCUSSION THREAD

**Discussion by tariqmahm** [ltsp/ltsp#232] [Q&A]
Hi,
   how to add new app in LTSP client.
I installed new app in LTSP server but app does not show in LTSP client user.
please guide, if anyone have solution

**Comment by alkisg** [comment]
At that point you need to update the LTSP image, by running:

```shell
sudo ltsp image /
```

Then reboot the client.

**Reply by tichzvidzayi** [reply]
Is there a way of installing an app directly to the client image without having to install the app on the server using the chroot mode?

**Comment by alkisg** [comment]
If you mean "is there any way to install an application inside the .squashfs image", the answer is no, it's not writable.

If you mean to install an application to a client while it's running, as a test, you can use sudo, and it will be lost on reboot.

If you mean something else, please clarify.

**Reply by tichzvidzayi** [reply]
Hi thanks for the quick reply, I mean for example installing LibreOffice so that clients can access it with having to install it on the server machine. The idea is to reduce applications on the server machine especially the ones that are rarely used by the adminstartor

**Comment by alkisg** [comment]
In the chrootless method (ltsp image /), the client has the same applications as the server.

In the virtual machine image and the chroot methods, the client has different applications than the server.

You can read more details about these 3 methods in the [ltsp ipxe man page](https://ltsp.org/man/ltsp-image/#image-types). It sounds like you'd prefer to use a VM image.
